### PR TITLE
Document the device overall_status and overall_progress fields

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: patch
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -2432,6 +2432,12 @@ dashboardDeviceUrl = balena.models.device.getDashboardUrl('a44b544b8cc24d11b036c
 <a name="balena.models.device.getAll"></a>
 
 ##### device.getAll([options]) ⇒ <code>Promise</code>
+This method returns all devices that the current user can access.
+In order to have the following computed properties filled in the result
```
Comment (sradevski) [inline DOCUMENTATION.md]:
```suggestion
In order to have the following computed properties in the result
```

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -2432,6 +2432,12 @@ dashboardDeviceUrl = balena.models.device.getDashboardUrl('a44b544b8cc24d11b036c
...
+you have to explicitly define then in a `$select` in the extra options:
```
Comment (sradevski) [inline DOCUMENTATION.md]:
```suggestion
you have to explicitly define them in a `$select` in the extra options:
```

Comment (thgreasi) [reply DOCUMENTATION.md]: I also had backticks, instead of single quotes in the `$select`. I guess my eyes were getting tired.

Comment (sradevski) [reply DOCUMENTATION.md]: I'll just pre-approve in case I am not around.

**Pull request merged**